packages/forms/src/styled/file-list/StyledFile.ts

```diff
@@ -5,30 +5,104 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { rgba } from 'polished';
 import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { StyledFileClose } from './StyledFileClose';
 
 const COMPONENT_ID = 'forms.file';
 
+const colorStyles = (props: IStyledFileProps & ThemeProps<DefaultTheme>) => {
+  const borderColor = getColor('neutralHue', 300, props.theme);
+  const focusBorderColor = getColor('primaryHue', 600, props.theme);
+  const foregroundColor = getColor('neutralHue', 800, props.theme);
+  const boxShadow = `
+    ${props.focusInset ? 'inset' : ''}
+    ${props.theme.shadows.md(rgba(focusBorderColor!, 0.35))}`;
+
+  return `
+    border-color: ${borderColor};
+    color: ${foregroundColor};
+
+    &:focus {
+      box-shadow: ${boxShadow};
+      border-color: ${focusBorderColor};
+    }
+  `;
+};
+
+const sizeStyles = (props: IStyledFileProps & ThemeProps<DefaultTheme>) => {
+  const size = `${props.theme.space.base * (props.isCompact ? 7 : 10)}px`;
+  const spacing = `${props.theme.space.base * 3}px`;
+
+  return `
+    box-sizing: border-box;
+    border: ${props.theme.borders.sm};
+    border-radius: ${props.theme.borderRadii.md};
+    padding: 0 ${spacing};
+    height: ${size};
+    font-size: ${props.theme.fontSizes.md};
+
+    & > span {
+      width: 100%;
+    }
+
+    & > ${StyledFileClose} {
+      width: ${size};
+      height: ${size};
+      margin-${props.theme.rtl ? 'left' : 'right'}: -${spacing};
+    }
+  `;
+};
+
 interface IStyledFileProps {
-  isDragging?: boolean;
   isCompact?: boolean;
+  focusInset?: boolean;
 }
 
 export const StyledFile = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledFileProps>`
-  display: inline-flex;
+  display: flex;
```

Comment:
Previous `display` did not follow RFC- and design-specified block styling. For most Garden listing components, the containing block controls the display width.